tables/ro-g0.ctb

```diff
@@ -0,0 +1,345 @@
+# 
+# liblouis: Uncontracted braille table for Romanian.
+#
+# Copyright (C) 2023 by BAUM Engineering <liblouis@baum.ro>
+#
+# This file is part of liblouis.
+#
+# liblouis is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation, either version 2.1 of the
+# License, or (at your option) any later version.
+#
+# liblouis is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with liblouis. If not, see
+# <http://www.gnu.org/licenses/>.
+
+#-index-name: Romanian, uncontracted
+#-display-name: Romanian braille
+
+#+language: ro
+#+region: ro
+#+type: literary
+#+dots: 6
+#+contraction: no
+#+grade: 0
+#+direction:both
+
+# This table is based on the French uncontracted braille table and was created with the help of 
+# Gosa Tudor, Papp Levente, Trutiu Florin and Patru Aurel from BAUM Engineering Romania and has been tested and used
+# in multiple schools and associations.
```

Comment:
Unfortunately an official documentation for Romanian braille does not exist as there are no universal rules that apply. There are different rules based on different regions or schools.
This table was made by us in collaboration with a local blind teacher and we also distributed it for the last 2 years as an add-on user defined table to be used on Index braille embossers.
It could be that changes will be made in the future to this table for the reason that there are no universal rules. But I think this table is a good starting point to have.

Reply:
OK thanks. It is crucial that this information is included in the table and test files. Also, because you say that there are different rules based on different regions or schools, it would be good to specify which are the schools and associations that have used and tested the table.

By the way, resolving conversations ought to be an exclusive right for us maintainers. Conversations are a nice way to track what is left to do in a pull request, so we want to avoid conversations that have been resolved but not actually addressed.

Reply:
@BAUM-Engineering Can you help us fix this today?